src/wp-includes/class-wp-theme.php

```diff
@@ -423,13 +439,15 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 						esc_html( $_child->template )
 					)
 				);
+				$_child->is_block_theme();
 				$_child->cache_add(
 					'theme',
 					array(
 						'headers'    => $_child->headers,
 						'errors'     => $_child->errors,
 						'stylesheet' => $_child->stylesheet,
 						'template'   => $_child->template,
+						'block_theme' => $_child->block_theme,
```

Comment:
Nit-pick: You always have `block_theme` first in the list, so let's do that here too. Also, the alignment needs to be fixed.
```suggestion
						'block_theme' => $_child->block_theme,
						'headers'     => $_child->headers,
						'errors'      => $_child->errors,
						'stylesheet'  => $_child->stylesheet,
						'template'    => $_child->template,
```